packages/config/config/devices/0x0234/templates/logic_group_template.json

```diff
@@ -0,0 +1,474 @@
+				"value": 2
+			}
+		]
+	},
+	"dimmer_pushbutton": {
+		"$import": "../../templates/master_template.json#base_0-255_nounit",
+		"label": "Operating pushbutton(s) for dimmer",
+		"maxValue": 15,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "Disabled",
+				"value": 0
+			},
+			{
+				"label": "Push Button 1",
+				"value": 1
+			},
+			{
+				"label": "Push Button 2",
+				"value": 2
+			},
+			{
+				"label": "Push Button 3",
+				"value": 4
+			},
+			{
+				"label": "Push Button 4",
+				"value": 8
+			}
+		]
+	},
+	"dimming_duration": {
+		"$import": "../../templates/master_template.json#base_0-255_nounit",
+		"label": "Duration of dimming",
+		"defaultValue": 5,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "Immediately",
+				"value": 0
+			},
+			{
+				"label": "Duration in seconds",
+				"value": 127
+			},
+			{
+				"label": "Duration in minutes (minus 127) from 1 – 128 minutes, where 128 is 1 minute",
+				"value": 255
+			}
+		]
+	},
+	"duration_on_off": {
+		"$import": "../../templates/master_template.json#base_0-255_nounit",
+		"label": "Duration of on/off",
+		"defaultValue": 0,
+		"allowManualEntry": true,
+		"options": [
+			{
+				"label": "Immediately",
+				"value": 0
+			},
+			{
+				"label": "Time in seconds",
+				"value": 127
+			},
+			{
+				"label": "Time in minutes (minus 127) from 1 – 128 minutes, where 128 is 1 minute",
+				"value": 255
+			}
+		]
+	},
+	"dimmer_mode": {
+		"$import": "../../templates/master_template.json#base_0-255_nounit",
+		"label": "Dimmer mode",
+		"maxValue": 2,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "No dimming, only on/off (0%/100%)",
+				"value": 0
+			},
+			{
+				"label": "Trailing edge dimming",
+				"value": 1
+			},
+			{
+				"label": "Leading edge dimming",
+				"value": 2
+			}
+		]
+	},
+	"dimmer_min": {
+		"$import": "../../templates/master_template.json#base_0-99_nounit",
+		"label": "Dimmer minimum level",
+		"unit": "%",
+		"defaultValue": 0
+	},
+	"dimmer_max": {
+		"$import": "../../templates/master_template.json#base_0-99_nounit",
+		"label": "Dimmer maximum level",
+		"unit": "%",
+		"defaultValue": 99
+	},
+	"led_indicator": {
+		"$import": "../../templates/master_template.json#base_0-255_nounit",
+		"valueSize": 4,
+		"unsigned": true,
+		"allowManualEntry": true
+	},
+	"pushbutton_led_indicator": {
+		"$import": "../../templates/master_template.json#base_0-255_nounit",
+		"label": "Pushbutton LED indicator",
+		"maxValue": 7,
+		"defaultValue": 7,
+		"unsigned": true,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "Internal LED disabled, only external commands control the indication",
+				"value": 0
+			},
+			{
+				"label": "LED follows switch",
+				"value": 1
+			},
+			{
+				"label": "LED follows switch, inverted",
+				"value": 2
+			},
+			{
+				"label": "Same as 1",
+				"value": 3
+			},
+			{
+				"label": "Same as 2",
+				"value": 4
+			},
```

Comment:
UIs probably show the label, not the value. So these labels aren't terribly helpful

Reply:
resolved